src/app/modules/twilio/twilio.controller.ts

```diff
@@ -54,6 +54,12 @@ export const twilioSmsUpdates: ControllerHandler<
    * Example: https://www.twilio.com/docs/usage/webhooks/sms-webhooks.
    */
 
+  // Extract public sender's ip address which was passed to twilio as a query param in the status callback
+  const url = req.url.split('?')
```

Comment:
hmm is it necessary to do this manually? can't we use the NodeJS [URL API](https://nodejs.org/api/url.html)?

Reply:
thanks! have done so